comfy_extras/nodes_mask.py

```diff
@@ -237,24 +237,24 @@ def combine(self, destination, source, x, y, operation):
         source = source.reshape((-1, source.shape[-2], source.shape[-1]))
 
         left, top = (x, y,)
-        right, bottom = (min(left + source.shape[-1], destination.shape[-1]), min(top + source.shape[-2], destination.shape[-2]))
+        right, bottom = (min(left + source.shape[-2], destination.shape[-2]), min(top + source.shape[-1], destination.shape[-1]))
         visible_width, visible_height = (right - left, bottom - top,)
 
         source_portion = source[:visible_height, :visible_width]
         destination_portion = destination[top:bottom, left:right]
```

Comment:
The above two lines are slicing along the wrong dimensions, should be:
```python
        source_portion = source[:, :visible_height, :visible_width]
        destination_portion = destination[:, top:bottom, left:right]
````
That's all I changed locally to fix the issue.

Reply:
Good find! I'll make the change. I wonder why both output shapes work (H,W) and (W,H)